### PR TITLE
Reformat README

### PR DESCRIPTION
**Pull request by lipis**
- Removed HTML tags for links, images.
- Correct language for each code snippet.
- Minor other styling fixes.

See it in action: [README.md](https://github.com/lipis/gae_mini_profiler/blob/reformat-readme/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,103 +4,120 @@ gae_mini_profiler is a quick drop-in WSGI app that provides ubiquitous profiling
...
-        
-        webapp2_extras.jinja2.default_config = {
-            "globals": {
-                "profiler_includes": gae_mini_profiler.templatetags.profiler_includes
-            }
-        }
-                
-    Using anything else to generate your HTML?
-        Just find some way to spit the results of profiler_includes() into your HTML. Doesn't have to be anything fancy.
+  There is a profiler_includes() function in gae_mini_profiler.templatetags that spits out the right code for these scripts and stylesheets.
```
Comment (benjaminjkraft) [inline README.md]: Those functions should be in backticks, I believe.

File: README.md [hunk 2]
```diff
@@ -110,23 +127,25 @@ profiler (below).</strong><br/>
...
 
-## <a name="faq">FAQ</a>
+## FAQ
+
+1. What's the license? [MIT licensed](http://en.wikipedia.org/wiki/MIT_License). There is also code from the Chromium project's DevTools whose license is at `static/chrome/inspector/devtools.html`.
+2. I had my `appstats_RECORD_FRACTION` variable set to 0.1, which means only 10% of my queries were getting profiles generated. This meant that most of the time gae_mini_profiler was failing with a javascript error, because the appstats variable was null.
 
-1. What's the license? [MIT licensed](http://en.wikipedia.org/wiki/MIT_License). There is also code from the Chromium project's DevTools whose license is at static/chrome/inspector/devtools.html.
-2. I had my appstats_RECORD_FRACTION variable set to 0.1, which means only 10% of my queries were getting profiles generated.  This meant that most of the time gae_mini_profiler was failing with a javascript error, because the appstats variable was null.
+  If you are using `appengine_config.py` to customize Appstats behavior you should add this to the top of your `appstats_should_record` method.
```
Comment (benjaminjkraft) [inline README.md]: This should probably get a number too.

**Comment by benjaminjkraft** [comment]
Thanks for the cleanup!  I noticed a few minor things you missed, so if you don't mind adding them it would be much appreciated.

Also, if you haven't yet, please sign our [CLA](http://www.khanacademy.org/r/cla).  (The bot to automatically tell you to do so should be set up soon!)

**Comment by lipis** [comment]
done and done.. :)

Maybe you guys could update the actual screenshots though in a later stage! :)

**Comment by benjaminjkraft** [comment]
Thanks.  And yep, those need updating -- just made #79 to track that.

**Pull request merged**